src/nbl/video/IOpenGL_LogicalDevice.h

```diff
@@ -81,6 +83,7 @@ class IOpenGL_LogicalDeviceBase
             ERT_INVALIDATE_MAPPED_MEMORY_RANGES,
             ERT_MAP_BUFFER_RANGE,
             ERT_UNMAP_BUFFER,
+            ERT_ALLOCATE,
```

Comment:
OpenGL can't allocate shit, no request needed

Reply:
discussed in discord